### ISSUE TITLE
create separate bundles for external modules

### ISSUE THREAD

**Issue by vigneshshanmugam** [elastic/synthetics#596]
+ With the support of bundling external packages, we bundle the included packages along with the source journey files to be able to run them via Heartbeat later without installing any packages. This is good, but it changes the source line numbers and column numbers of those particular journeys and might be a bad UX if we have to go back to the original source. 

More details in here - https://github.com/elastic/synthetics/pull/580#pullrequestreview-1082955330

The ultimate goal of our push command is to replicate exactly how your folder structure is on your local and run the same inside HB when pushed. This would make it presentable in Kibana when we add unzipping support and allow users to view and edit monitors.

TLDR: If i push below example journey code 
~~examples/bundled-code/test.journey.ts~~
```ts
import { journey, step, monitor, expect } from "@elastic/synthetics";
import isPositive from "is-positive";
import leftpad from "leftpad";
import * as utils from "./utils";

journey("example journey", ({ page, params }) => {
  step("example step", () => {});
});
```

Our bundled zip folder should be in the below format. 

```
examples/bundled-code/test.journey.ts
examples/bundled-code/utils.ts
examples/bundled-code/is-positive.js
examples/bundled-code/leftpad.js
```

**Comment by vigneshshanmugam** [comment]
We have a way to create the separate file for each journey file and utils without recreating the files during bundling instead using the source-maps to recover the file structure in Kibana if required in the future. The error stack trace is fixed via the PR - #687, but since we dont have a UI in Kibana to showcase the file structure, we dont need to solve this issue for now and have it in our backlog.
